skstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -0,0 +1,342 @@
+import numpy as np
+from scipy.stats import norm
+
+from .basecalculator import BaseCalculator
+from ..fitutils.utils import eval_pdf, array2dataset, pll
+from ..parameters import POI
+
+
+def generate_asimov_hist(model, params, nbins=100):
+    """ Generate the Asimov histogram using a model and dictionary of parameters.
+
+        Args:
+            model : model used to generate the dataset
+            params (Dict) : values of the paramters of the models
+            nbins (int, optionnal) : number of bins
+
+        Returns:
+             (`np.array`, `np.array`) : hist, bin_edges
+
+        Example:
```

Comment:
I think this kind of example should explicitly say that you are here using zfit as a backend, but any other backend following the same API would work. This is probably a comment to put at high-level somewhere too?

Reply:
I agree, for now all the examples are using zfit as backend,  so I could replace "Example" -> "Example with fit" ?

Reply:
Guess so :-).

Reply:
The general comment on backends is still valid, as you spent some time on the API ;-).

Reply:
@eduardo-rodrigues what do you mean?

Reply:
I mean that you are providing examples with zfit but another package with the same API for models, etc., would also work. Correct? This is important info about the package, and should be written down on the README and probably also in some examples or modules.

Reply:
Absolutely :)